src/provider.ts

```diff
@@ -25,6 +25,12 @@ type ProviderOptions = {
   provider: Record<string, Provider>
 }
 
+type UtilitiesProviderOptions = {
```

Comment:
rename to `ProviderUtilityOptions` - this orders the concepts in importance

Reply:
Ok